### DISCUSSION TITLE
Where is Sapphire layout?

### DISCUSSION THREAD

**Discussion by vitsin** [primefaces/community#1521] [PrimeFaces Templates] [answered]
hello,
https://www.primefaces.org/sapphire/ - is http 404
Sapphire is discontinued or URL is simply broken?

thanks,
--vs

**Comment by melloware** [comment]
Looks like all the premium template sites are not available except the 5 latest ones. PrimeTek needs to chime in with what the plan is.

**Reply by rider87** [reply]
what happened with all the premium templates? I paid a lot of money for using them!!!!!!  

your text (https://www.primefaces.org/blog/unleash-the-power-of-primefaces-12-with-our-updated-premium-templates):
"By using one of our premium templates, you can save yourself a lot of time and effort when building your next web project. The templates are fully customizable, so you can easily make them match your brand and add your own unique flair."

If those templates are not supported anymore, we will not save time!!  
I hope this is a joke!

This is why I bought a few templates, also mentioned on your side:
https://www.primefaces.org/layouts/licenses

<img width="965" alt="image" src="https://github.com/primefaces/community/assets/9787533/c352a247-ec81-48d1-b3c4-50d6d42d6810">

![image](https://github.com/primefaces/community/assets/9787533/9b7f1442-eb3a-4068-a03a-ac0c58ce988d)

**Comment by onursenture** [comment, marked as answer]
Hello, you can access the official statement on this matter at the following address: https://github.com/orgs/primefaces/discussions/1806

**Comment by tandraschko** [comment]
we decided to only maintain
          - "apollo"
          - "atlantis"
          - "avalon"
          - "diamond"
          - "freya"
          - "poseidon"
          - "ultima"
          - "verona"
          
for now
